code-of-conduct.md

```diff
@@ -0,0 +1,105 @@
+
+## Our standards
+
+Examples of behavior that contributes to creating a positive environment for the Ballerina community include:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the overall community
+
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery, and sexual attention or advances of any kind
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others’ private information, such as a physical or email address, without their explicit permission
+- Other conduct, which could reasonably be considered inappropriate in a professional setting
+
+## Our responsibilities
+
+Project maintainers are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this code of conduct, and will communicate reasons for moderation decisions when appropriate.
+
+## Scope
+
+This code of conduct applies both within project spaces and in public spaces, and also applies when an individual is officially representing the project or its community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the project team responsible for enforcement at **legal@wso2.com**. All complaints will be reviewed and investigated promptly and fairly.
+
+All project/community leaders are obligated to respect the privacy and security of the reporter of any incident.
+
+## Enforcement guidelines
+
+Project/Community leaders will follow the below Community Impact Guidelines in determining the consequences for any action they deem in violation of this code of conduct:
+
+**1. Correction**
+
+<table border=1>
+<th> Community Impact </th>
+<th> Consequence </th>
+<tr border=1>
+<td width=50% border=1> Use of inappropriate language or other behavior deemed unprofessional or unwelcome in the community. </td>
+<td width=50% border=1> A private, written warning from community leaders providing clarity around the nature of the violation and an explanation of why the behavior was inappropriate. A public apology may be requested.
```

Comment:
Should we change this also to project/community leaders?

Reply:
Fixed.